### ISSUE TITLE
Auto exporting of dascript functions

### ISSUE THREAD

**Issue by taylorius** [GaijinEntertainment/daScript#91]
Is there a way to make a dascript function visible from C code, without needing the [export] line? Perhaps an option to make all functions visible by default?

**Comment by borisbat** [comment]
options remove_unused_symbols=false does that
the implication is that everything in the context would be exported, even though it may not be used

**Comment by devilinthedata** [comment]
Thank you very much, borisbat! That does indeed work. However I just realised my question was not specific enough. I was hoping for a way of achieving this from the C++ side, without relying on anything in the script. I think I've just spotted a parameter to the compileDaScript function which does what I need - "bool exportAll=false". Setting this to true seems to export the symbols without any script commands.

Thanks so much for your help on this, and on the other problem I posted. DaScript is exactly what I've been looking for for a long time, and I'm really excited to see the things it can do!

**Comment by taylorius** [comment]
Thank you very much, borisbat! That does indeed work. However I just realised my question was not specific enough. I was hoping for a way of achieving this from the C++ side, without relying on anything in the script. I think I've just spotted a parameter to the compileDaScript function which does what I need - "bool exportAll=false". Setting this to true seems to export the symbols without any script commands.

Thanks so much for your help on this, and on the other problem I posted. DaScript is exactly what I've been looking for for a long time, and I'm really excited to see the things it can do!
